### ISSUE TITLE
Nest 6 and app.disable('x-powered-by')

### ISSUE THREAD

**Issue by andreialecu** [nestjs/nest#1788]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

With Nest 6, what is the proper way to disable the `x-powered-by: Express` header? It seems like `disable(...)` has been removed.

I don't see any relevant information in the v6 documentation about this particular change.

## What is the motivation / use case for changing the behavior?

```typescript
async function bootstrap() {
  const app = await NestFactory.create(AppModule);
  app.disable('x-powered-by'); // no longer works
  app.use(nocache());
  await app.listen(1338);
}
bootstrap();
```

## Environment

<pre><code>
Nest version: 6.0.1
</code></pre>

**Comment by kamilmysliwiec** [comment]
```typescript
const app = await NestFactory.create<NestExpressApplication>(AppModule);
app.disable('x-powered-by');
```
`NestExpressApplication` is imported from `@nestjs/platform-express` package.
see https://docs.nestjs.com/first-steps#platform

**Comment by andreialecu** [comment]
Thanks. I think the [migration guide](https://docs.nestjs.com/migration-guide) could use this information as well.

**Comment by sajidali** [comment]
@kamilmysliwiec interestingly when I tried your suggestion with nest 6.2.4, I am unable to properly remove `x-powered-by: Express` header. when I add `app.disable('x-powered-by');` and check the headers again, the header is changed to capitalised word version `X-Powered-By`.

I have tried adding both casings as:
app.disable('x-powered-by');
app.disable('X-Powered-By');

but api is still responding with `X-Powered-By: Express`.
